cli/lib/kontena/cli/stacks/labels/list_command.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Labels
+  class ListCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    parameter "NAME", "Stack name"
+
+    requires_current_master
+    requires_current_master_token
+
+    def execute
+      stack = client.get("stacks/#{current_grid}/#{name}")
+      # safeguard from nil, since 'labels' field is optional
+      puts Array(stack['labels'] || []).join("\n")
```

Comment:
See the table generator thing on other `list_command`s

Reply:
@kke not sure if that is comparable to things like `kontena stack ls` or `kontena service ls`... I took the inspiration from `kontena node labels ls <node name>`:
```ruby
def execute
  node = client.get("nodes/#{current_grid}/#{self.node}")
  puts Array(node['labels']).join("\n")
end
```
If you feel the output should be table-alike, could you suggest an example?

Reply:
Hmm, indeed. I need to look into this at office hours, I don't know why `node labels ls` is not a regular list command. Maybe @jakolehm has an opinion about this?